misc/testerina/modules/testerina-core/src/main/ballerina/Module.md

```diff
@@ -1,27 +1,40 @@
 ## Module overview
 
-This module facilitates developers to write automation tests for Ballerina code in a simple manner. It provides a number of capabilities such as configuring setup and cleanup steps in different levels, ordering and grouping of tests, providing value-sets to tests and independence from external functions and endpoints via mocking capabilities.
+This module facilitates developers to write automation tests for Ballerina code in a simple manner. 
+It provides a number of capabilities such as configuring setup and cleanup steps in different 
+levels, ordering and grouping of tests, providing value-sets to tests and independence from external
+functions and endpoints via mocking capabilities.
 
 ## Annotations
-A Ballerina testsuite can be implemented using a set of annotations. The available annotations enable executing instructions before and after the testsuite or a single test, organize a set of tests into a group, define data-driven tests, specify an order of execution, disable tests and mocking.
+A Ballerina testsuite can be implemented using a set of annotations. The available annotations 
+enable executing instructions before and after the testsuite or a single test, organize a set of 
+tests into a group, define data-driven tests, specify an order of execution, disable tests and 
+mocking.
+
+### Test Config 
 
 The following example shows a simple testsuite.
 ```ballerina
-import ballerina/io;
 import ballerina/test;
 
-// Before Suite Function
-@test:BeforeSuite
-function beforeSuiteFunc() {
-    io:println("I'm the before suite function!");
+// Test function
+@test:Config {}
+function testFunction() {
+    test:assertTrue(true, msg = "Failed!");
 }
+```
+
+The test config annotation can be used to set the test execution order by specifying the before and 
```

Comment:
It should be The `before` and `after` attributes, right? To be consistent with the details you've provided about other attributes.

Reply:
Made it consistent with the other details